kobo/apps/audit_log/models.py

```diff
@@ -48,6 +57,7 @@ class AuditLog(models.Model):
         db_index=True
     )
     user_uid = models.CharField(db_index=True, max_length=UUID_LENGTH + 1)  # 1 is prefix length
+    log_type = models.CharField(choices=AuditType.choices)
```

Comment:
is it a criteria we are going to (often) use to narrow down query? I think I would use `db_index=True`? 
What do you think?